src/main.ts

```diff
@@ -784,16 +828,20 @@ function generateToJson(
           from,
           defaultValue(field.type, options)
         );
-      } else {
```

Comment:
Collapsing this `else` statement into an `else if` made this diff kind of hard to read. :-/